README.md

```diff
@@ -1,15 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+To Do list: list structure
 
 > Description the project.
+Building a simple HTML list of To Do tasks using webpack and served by a webpack dev server.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Javascript
 
 ## Live Demo (if available)
 
```

Comment:
- [ ] Kindly customize the README file for each project you start. Use the correct project name, provide the instructions for a new user to install and use the app with the `npm` commands, if there is no 'Live Demo' don't use a link, etc.